OffscreenCanvasAnimation.md

```diff
@@ -214,20 +215,38 @@ interface HTMLVideoElementFrameRequestData : FrameRequestData {
 }
 ```
 
-#### Issues
-* Does not support the use case of a continuously running task that produces frames repeatedly over time.  Not supporting this makes it inconvienient to port native application to the Web (e.g. via Emscripten) because the apps are often not deisigned for an asynchronous mode of operation.
+### OffscreenCanvas commit
+
+```
+interface OffscreenRenderingContext {
+  void commit();
+  readonly attribute OffscreenCanvas canvas;
+};
+
+interface OffscreenCanvasRenderingContext2D : OffscreenRenderingContext {};
+interface OffscreenWebGLRenderingContext : OffscreenRenderingContext, WebGLRenderingContext {};
+```
+
+#### The commit processing model
+
+When `commit()` is called, a copy of the frame is made and sent to the driver/display when appropriate. This function can return immediately or block. The UA can decide to queue a few frames before blocking to guarantee smoothness.
+
+
+#### Rationale
+* RAF allows for animations to be synced and support current rendering models on the web.
+* commit() supports the use case of a continuously running task that produces frames repeatedly over time, wihch enables the port of native application to the Web (e.g. via Emscripten).
```

Comment:
wihch -> which

Reply:
I think the context here could be developped a bit more. Should mention WASM, and how following an async model requires exiting and re-entering the WASM scope, which is bad.